Zend/zend.c

```diff
@@ -522,6 +522,7 @@ static void zend_set_default_compile_time_values(void) /* {{{ */
 {
 	/* default compile-time values */
 	CG(short_tags) = short_tags_default;
+	CG(short_tags_notice) = IS_FALSE;
```

Comment:
`CG(short_tags_notice)` is declared as `zend_bool`, and as such the values `0` (false) and `1` (true) are customary.  `IS_FALSE` and `IS_TRUE`, on the other hand, are type tags used for `zval`s ([details](http://www.phpinternalsbook.com/php7/internal_types/zvals/basic_structure.html#types-and-values)). These should not be mixed up.

Reply:
Fixed in 576408b56b660791f419a1a66e51b06f65266d3b